lib/trie/branch/type.go

```diff
@@ -0,0 +1,12 @@
+package branch
```

Comment:
could we add a license here?

Reply:
done :+1: Note this is in the CI so it will ultimately not allow to merge without it :wink: